src/MatrixTypes_PETSc.f90

```diff
@@ -193,7 +193,6 @@ SUBROUTINE init_PETScMatrixParam(matrix,Params)
 !> @param matrix the matrix type to act on
 !>
 SUBROUTINE clear_PETScMatrixType(matrix)
-  CHARACTER(LEN=*),PARAMETER :: myName='clear_PETScMatrixType'
```

Comment:
Why are all of these changed?

Reply:
Travis marked them as unused parameters, and the branch failed the check. I assumed there was a change made to Travis around the time whitespace was merged in.

Reply:
Fair point... but concerning we don't have a single error message for any of these routines or check ierr coming from PETSc.

Reply:
Agreed, but such a check could potentially add a lot of overhead when running these functions (which happens a lot moving data between petsc for CMFD and the native MOC).

I don't particularly like removing these. I suppose the "unused-parameter" check could be removed from the CI checks, but that's the only alternative I see. It would still detect unused variables, but parameters like the routine names would be ignored.

Reply:
Would be worth investigating some DBC statements.  That way no overhead is introduced in release builds but we still have at least some error checking.

Reply:
I agree 100% and was thinking about that during my drive in.  But its WAY outside the scope of this ticket (and would still remove the "myName" parameter.  In general, we need to go through a lot of the code and consider using DBC better.